tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -23,18 +30,7 @@ parameters:
 - name: officeFeedUrl
   type: string
 
-# If the template should try to checkout the repo. If the job that includes this template already has steps to check out
-# the repo for other purposes, this should be set to false.
-- name: isCheckoutNeeded
-  type: boolean
-  default: false
```

Comment:
Was relevant for pipelines that did not check out the repo for their functionality (e.g. running e2e tests) but needed to do it in order to get access to telemetry-generator. Now they install it from an internal ADO feed so they don't need to check out the repo any more.